aspnetcore/migration/mvc/sample/WebApp1.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="wwwroot\css\site.css" />
+    <Content Include="wwwroot\css\site.min.css" />
+    <Content Include="wwwroot\favicon.ico" />
+    <Content Include="wwwroot\js\site.js" />
+    <Content Include="wwwroot\js\site.min.js" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="BundlerMinifier.Core" Version="2.3.327" />
```

Comment:
@tdykstra It may be best to remove this NuGet package, since it really isn't being used for anything in the sample code. To @spboyer's point, there are no hooks such as PrePublish that rely on it.